modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,169 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift Container Platform cluster that is running on ARM hardware.
++
+include::snippets/technology-preview.adoc[]
 
-With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift Container Platform cluster that is running on ARM hardware.
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
+// (link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
 
-include::snippets/technology-preview.adoc[]
+* Previously, as a developer, you could run only one pipeline run from your `.tekton` directory for each Git event, such as a pull-request. With this update, you can run all the pipeline runs in your `.tekton` directory that match specified events. The web console displays the status and reports of the runs. If you are using WebVCS, the reports include comments.
+// (link:https://issues.redhat.com/browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, you can install and run a Tekton Hub instance with minimal configuration. This way, you can start working with your teams to decide which additional customizations they might want.
+// (link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+* Getting notifications if you add a duplicate repository. With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you use`tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
+// (link:https://issues.redhat.com/browse/SRVKP-1758[SRVKP-1758 Pipeline as code: validate repository URL uniqueness])
+// Shivam Mukhade
+
+* Using a custom database with Tekton Hub. With this update, as an administrator, you can configure your local Tekton Hub instance to use a custom database rather than the default database.
+// (link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* Automatically refreshing the Tekton Hub catalog and database. With this update, as a cluster administrator, if you enable your local Tekton Hub instance, it has a default cron job that periodically refreshes the database so that changes in the catalog appear in the Tekton Hub web console. You can adjust the period between refreshes.
```

Comment:
its not cron job, its a built-in functionality inside code

Reply:
Thanks, updated.

Reply:
Updated. Thanks.